index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Modern eCommerce Landing Page</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <!-- Header Section -->
+    <header>
+      <nav>
+        <h1>Shop<span>Gen</span></h1>
+        <ul class="nav-links">
+          <li><a href="index.html">Home</a></li>
+          <li><a href="#products">Shop</a></li>
+          <li><a href="checkout.html">Checkout</a></li>
+          <li><a href="#about">About</a></li>
+          <li><a href="#contact">Contact</a></li>
+        </ul>
+        <div class="cart" id="cart">Cart (0)</div>
+      </nav>
+    </header>
+
+    <!-- Hero Section -->
+    <section class="hero" id="home">
+      <div class="hero-text">
+        <h2>Find Your Style</h2>
+        <p>
+          Discover exclusive collections and the latest trends. Get them before
+          they're gone!
+        </p>
+        <a href="#products" class="btn">Shop Now</a>
+      </div>
+      <div class="hero-img">
+        <img src="images/shoe.webp" alt="Hero Image" />
+      </div>
+    </section>
+
+    <!-- Product Section -->
+    <section id="products" class="products">
+      <h3>Featured Products</h3>
+      <div class="product-gallery" id="product-gallery">
+        <!-- Products will be injected by JS -->
+      </div>
+    </section>
+
+    <!-- Footer Section -->
+    <footer>
+      <p>&copy; 2024 ShopGen. All Rights Reserved.</p>
+    </footer>
+
+    <script src="script.j"></script>
```

Comment:
This is a job well done, however your Script file is missing
I suggest you add a JavaScript file